### PR TITLE
Fix ClusterMachineApproverDown alert

### PR DESCRIPTION
**Pull request by spangenberg**

### CODE REVIEW

**Comment by enxebre** [comment]
can you please enable the `e2e-aws` suite here first so this will block unless working as expected?

File: manifests/03-metrics-service.yaml [hunk 1]
```diff
@@ -12,8 +12,9 @@ spec:
   clusterIP: None
   ports:
   - name: https
-    port: 9192
-    targetPort: https
+    protocol: TCP
```
Comment (enxebre) [inline manifests/03-metrics-service.yaml]: why?

Comment (enxebre) [reply manifests/03-metrics-service.yaml]: metrics are already being reported https://xlugntax-promecieus.svc.ci.openshift.org/graph?g0.range_input=1w&g0.stacked=1&g0.moment_input=2019-10-15%2014%3A55%3A54&g0.expr=mapi_max_pending_csr&g0.tab=0

Comment (spangenberg) [reply manifests/03-metrics-service.yaml]: Thanks, confused the named port.

**Pull request merged**